pkg/fanal/analyzer/language/rust/cargo/cargo_test.go

```diff
@@ -382,6 +382,133 @@ func Test_cargoAnalyzer_Analyze(t *testing.T) {
 			dir:  "testdata/sad",
 			want: &analyzer.AnalysisResult{},
 		},
+		{
+			name: "workspace members",
+			dir:  "testdata/toml-workspace-members",
+			want: &analyzer.AnalysisResult{
+				Applications: []types.Application{
+					{
+						Type:     types.Cargo,
+						FilePath: "Cargo.lock",
+						Libraries: types.Packages{
+							{
+								ID:       "aho-corasick@1.1.2",
+								Name:     "aho-corasick",
+								Version:  "1.1.2",
+								Indirect: true,
+								Locations: []types.Location{
+									{
+										StartLine: 5,
+										EndLine:   12,
+									},
+								},
+								DependsOn: []string{"memchr@2.6.4"},
+							},
+							{
+								ID:       "gdb-command@0.7.6",
+								Name:     "gdb-command",
+								Version:  "0.7.6",
+								Indirect: false,
+								Locations: []types.Location{
+									{
+										StartLine: 14,
+										EndLine:   22,
+									},
+								},
+								DependsOn: []string{
+									"regex@1.10.2",
+									"wait-timeout@0.2.0",
+								},
+							},
+							{
+								ID:       "libc@0.2.150",
+								Name:     "libc",
+								Version:  "0.2.150",
+								Indirect: true,
+								Locations: []types.Location{
+									{
+										StartLine: 24,
+										EndLine:   28,
+									},
+								},
+							},
+							{
+								ID:       "memchr@2.6.4",
+								Name:     "memchr",
+								Version:  "2.6.4",
+								Indirect: true,
+								Locations: []types.Location{
+									{
+										StartLine: 44,
+										EndLine:   48,
+									},
+								},
+							},
+							{
+								ID:       "regex@1.10.2",
+								Name:     "regex",
+								Version:  "1.10.2",
+								Indirect: true,
```

Comment:
Shouldn't it be false? It is defined in Cargo.toml.

Reply:
Fixed.